services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/modelsTable.tsx

```diff
@@ -54,7 +96,7 @@ const ModelsTable = ({ activityId, prompt }) => {
     { name: "Created At", attribute:"created_at", width: "100px" },
     { name: "Version", attribute:"version", width: "70px" },
     { name: "Model Name", attribute:"name", width: "300px" },
-    // { name: "Model Notes", attribute:"", width: "200px" },
+    { name: "Model Notes", attribute:"notes", width: "150px" },
```

Comment:
I added a somewhat aribtary width value that seemed reasonable.

Reply:
yeah-- totally fine!